packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -208,7 +208,8 @@ export default class Meetings extends WebexPlugin {
 
       if (data.locus && data.locus.replaces && data.locus.replaces.length > 0) {
         // Findes the first occurence of the meeting in the collection with same locus url
-        meeting = data.locus.replaces.some((replacedLocus) => this.meetingCollection.getByKey(LOCUS_URL, replacedLocus.locusUrl));
+        // Always the last element in the replace is the active one
+        meeting = this.meetingCollection.getByKey(LOCUS_URL, data.locus.replaces[data.locus.replaces.length - 1].locusUrl);
```

Comment:
Was this supposed to be in this PR? This doesn't seem to be part of the meeting upload feature.

Reply:
There was a big which is related to replace locus when user land on different locus node, I can split it out to a separate PR

Reply:
done